app/brave_main_delegate.cc

```diff
@@ -212,6 +213,7 @@ bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
 #if defined(OS_WIN)
       features::kWinrtGeolocationImplementation.name,
 #endif
+      security_state::features::kSafetyTipUI.name,
```

Comment:
It would be good to add test here(https://github.com/brave/brave-core/blob/master/app/brave_main_delegate_browsertest.cc#L91) for this enabling.